.github/workflows/continuous-integration-workflow.yml

```diff
@@ -79,23 +108,27 @@ jobs:
         if: ${{ matrix.cxx == 'clang++' && startsWith(matrix.distro,'fedora:') }}
         run: echo "FC=flang-new" >> $GITHUB_ENV
       - name: maybe_use_external_gtest
-        if: ${{ matrix.distro == 'ubuntu:latest' }}
+        if: ${{ matrix.distro == 'ubuntu:latest' && matrix.cxx_extra_flags != '-m32 -DKOKKOS_IMPL_32BIT' }}
```

Comment:
```suggestion
        if: ${{ matrix.distro == 'ubuntu:latest' && matrix.cxx_extra_flags != '-m32' }}
```

Reply:
I see.  So this is sidestepping the configuration-time setting.  Not sure how much I like it but I don't have a better idea just yet.